http/headers/feature-policy.json

```diff
@@ -1333,12 +1432,64 @@
               "edge": {
                 "version_added": false
               },
-              "firefox": {
-                "version_added": "92"
-              },
-              "firefox_android": {
-                "version_added": "92"
-              },
+              "firefox": [
+                {
+                  "version_added": "92",
+                  "partial_implementation": true,
+                  "notes": "Only supported through the <code>allow</code> attribute on <code>&lt;iframe&gt;</code> elements.",
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "media.setsinkid.enabled",
+                      "value_to_set": "true"
+                    }
+                  ]
+                },
```

Comment:
I'm not sure if this partial implementation entry should exist until the `media.setsinkid.enabled` flag is enabled by default in Firefox? While this feature policy is recognised by Firefox without this flag, it seems pointless to mark as supported without it having any noticeable effect.

Reply:
I think the way you've structured this is a little unusual. Partial + flags isn't terribly common, but it comes up from time to time. What I'm confused by is how it relates to the support statement immediately following. Since neither statement has a `version_removed`, it seems to say that it's partially supported with only one flag set and fully supported with both?

I think the right move here might be one support statement with the note and partial from above and the array of flags from below. Does that make sense?

Reply:
The problem it's trying to solve is that the feature policy is supported in header form with two flags enabled. But only on the iframe (see notes) without one of the extra flags set. So I'm not sure the one entry would make much sense. I'm thinking maybe just keep the top entry and remove the bottom one, as the second flag isn't overly useful?

Reply:
> It would seem that Firefox don't intend to enable feature policy header (it's been superseded by permissions policy) hence my thinking that the flag might not be too useful.

Yes, that's a strong point in favor of not mentioning the other case. I'd drop the second statement, in that case.